bench/app.py

```diff
@@ -123,16 +116,14 @@ def url(self):
 		if self.on_disk:
 			return os.path.abspath(self.name)
 
-		if self.use_ssh:
-			return self.get_ssh_url()
+		if self.from_name_tag:
+			return self.get_http_url()
 
-		return self.get_http_url()
+		# return the git url as is
+		return self.name
 
 	def get_http_url(self):
-		return f"https://{self.remote_server}/{self.org}/{self.repo}.git"
-
-	def get_ssh_url(self):
-		return f"git@{self.remote_server}:{self.org}/{self.repo}.git"
+		return f"https://github.com/{self.org}/{self.repo}.git"
```

Comment:
Doesn't this also assume `github.com` as the host? If you store the parsed URL in the `App` instance, that'll let you access the host name too.

Reply:
So the `get_http_url` method will only be called in the case when `self.from_name_tag` is `True` i.e the cases where user provides ONLY the repo name and/or tag/branch as in these cases we check if the repos belong under frappe/erpnext org (eg: `bench get-app healthcare` or `bench get-app healthcare@develop`) - in rest of the cases where the user provides a (valid) git url, it will just be returned as is

Reply:
Oh, I see now. In that case, this is fine (you might want to keep the `remote_server` variable and just use that though).

We could also later consider receiving an optional parameter for the host (defaults to github):

`bench get-app gitlab org/repo`

Reply:
i read the logic for `[fetch_details_from_tag](https://github.com/frappe/bench/blob/develop/bench/utils/__init__.py#L403)` method and i think it can do it for all the repos on github (atleast for now) - if the format is `bench get-app org/repo@branch` ..i'm shocked 😅 

also, if i do something like `bench get-app org/repo` the git parser picks it up to be a valid url 🙄 - which errors out 😓